lib/Widgets/AbstractSimpleSettingsPage.vala

```diff
@@ -96,12 +98,10 @@ public abstract class Granite.SimpleSettingsPage : Granite.SettingsPage {
 
     /**
      * Creates a new SimpleSettingsPage
+     * Deprecated: Subclass this instead.
      */
     public SimpleSettingsPage () {
-        Object (activatable: activatable,
-                icon_name: icon_name,
-                description: description,
-                title: title);
+        
     }
 
```

Comment:
You cannot instantiate an abstract class anyway so this (empty) creator function should be removed. The preceding comments are redundant.  But why is this class abstract an abstract class?  It does not contain any abstract methods nor does its parents.

Reply:
Yeah I fully agree with you, but it's too late (or too early) in the cycle to introduce an API break so i'll keep it like this right now